rest-api/src/main/resources/definitions/participant_file.yml

```diff
@@ -0,0 +1,26 @@
+description: |
+    A descriptor of a participant blob file
+type: object
+properties:
+    userId:
+        type: string
+        x-nullable: false
+    fileId:
+        type: string
+    mimeType:
+        type: string
+        x-nullable: false
+    appId:
+        type: string
```

Comment:
You can remove appId from the API (both the server and the SDK)...it's always implied by the caller's session.